docs/pages/access-controls/access-requests/oss-role-requests.mdx

```diff
@@ -0,0 +1,159 @@
+```
+
+By default, this command will block until the request is approved. To submit the
+request without waiting for approval, add the `--nowait` flag.
+
+Alternatively, `tsh` can automatically create an access request during the login
+process. To activate this behavior, specify the `--request-roles` flag:
+
+```code
+$ tsh login --user=alice --request-roles=dba
+# Seeking request approval... (id: bc8ca931-fec9-4b15-9a6f-20c13c5641a9)
+```
+
+This will wait for the request to be approved, and then issue credentials with
+the `dba` role automatically when the request is approved.
+
+To log in and submit the request without waiting for approval, add the
+`--request-nowait` flag. In this scenario, you will receive your regular roles
+upon login, and can elevate your access after the request is approved.
+
+```code
+# log in with an approved access request
+$ tsh login --request-id=bc8ca931-fec9-4b15-9a6f-20c13c5641a9
+```
+
+You can list requests using `tsh request ls`.
+
+```code
+$ tsh request ls
+# Token                                Requestor Metadata       Created At (UTC)    Status
+# ------------------------------------ --------- -------------- ------------------- -------
+# bc8ca931-fec9-4b15-9a6f-20c13c5641a9 alice     roles=dba      07 Nov 19 19:38 UTC PENDING
+```
+
+## Reviewing requests
+
+In OSS Teleport, Access Requests must be reviewed by a cluster administrator
+with the ability to run `tctl` on the Auth Server.
+
+Administrators can list requests with `tctl requests ls`, and view the details
+of a particular request with `tctl requests get`.
+
+To quickly approve or deny a request, use `tctl requests review`.
+
+For example, to approve a request:
+
+```code
+$ tctl requests review --approve --author=bob bc8ca931-fec9-4b15-9a6f-20c13c5641a9
+```
+
+<Admonition type="warning">
+Granting access to a role with the ability to edit other roles could allow a
+user to **permanently** upgrade their privileges. When reviewing requests, it's
+important to carefully consider the role(s) being requested and what permissions
+will be applied to the user if the request is approved.
+</Admonition>
+
+To deny the request, replace `--approve` with `--deny`.
+
+Note the use of the `--author` flag - since `tctl` runs with a built-in
+administrator identity, this identifies the Teleport user who performed the
+review, which adds helpful context for the request submitter.
+
+If you require more control over the review process, you can use the
+`tctl requests approve` and `tctl requests deny` family of commands. These
+commands provide several additional features.
+
+These commands allow the reviewer to specify the reason for the approval/denial.
+For example:
+
+```code
+$ tctl request deny \
+    --reason="today's migration has been canceled" \
+    bc8ca931-fec9-4b15-9a6f-20c13c5641a9
+```
+
+Additionally, the reviewer can override the set of roles in the request:
+
+```code
+$ tctl request approve \
+    --roles="db-support" \
```

Comment:
To match the role specified above:

```suggestion
    --roles="dba" \
```

Reply:
Good catch!

Reply:
Oh, no. This is intentional. This example is showing how to override the role the user requested and replace it with a different one.